backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/views/GL_Journal_Balances_v.sql

```diff
@@ -0,0 +1,81 @@
+DROP VIEW IF EXISTS GL_Journal_Balances_v
+;
+
+CREATE OR REPLACE VIEW GL_Journal_Balances_v AS
+SELECT
+    /*This view is only for a tab in the GL_Journal window where the gl_journal_id is unique. Summing it up with the account ID will ensure uniqueness on window level.
+      If the view's purpose must be extended, another primary key formula must be applied!
+     */
+    t.gl_journal_id + t.account_id AS gl_journal_balances_v_id,
```

Comment:
> Summing it up with the account ID will ensure uniqueness on window level.

not really.
consider 
GL_Journal_ID=1000003, Account_ID=1000007
GL_Journal_ID=1000007, Account_ID=1000003
----

I suggest, 
* drop gl_journal_balances_v_id
* flag GL_Journal_ID as IsParent=Y
* flag Account_ID as IsParent=Y